usagov/usagov-2021
# USAGOV-810-Complaints-Icon

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-810

## Testing

* Edit a Navigation page that appears on the homepage
* In the CSS Icon field, enter: "icon-complaints"
* Check the homepage for the complaints icon

## Checklist for the Developer



## Checklist for the Peer Reviewers
